amplify/backend/function/expressServer/src/controllers/leagueController.js

```diff
@@ -11,20 +11,31 @@ const createLeague = async (req, res, next) => {
         // parse the body and destructure
         const {
             leagueName,
-            budget,
+            startingBalance,
             leagueType,
             tradingFee,
-            portfolios,
-            private
+            private,
+            startDate,
+            WinningValue,
```

Comment:
Should this be lower case? Just will be good to have unified so doesn't trip us up when sending data remembering that this is upper case.